src/make_app.ts

```diff
@@ -65,10 +70,17 @@ export function makeApp(options: AppOptions): PolyserveApplication {
 
     if (headers) {
       for (const header in headers) {
-        (<any>res).append(header, headers[header]);
+        (<any>res).setHeader(header, headers[header]);
       }
     }
-    send(req, filePath).pipe(res);
+
+    const _send = send(req, filePath, {
+      lastModified: false,
+      cacheControl: false,
+    });
+    // Uncomment this to disable disable 304s from send() and always compile
+    // _send.isFresh = () => false;
```

Comment:
This issue is that if the browser sends a cache-control header because it has a cached version of the resource, and `send()` determines that the resource hasn't been modified, the response will be a 304 and won't contain a body or mime-type, and thus we can't compile it. This is actually the desired behavior for users, but during development it was massively hard to tell what was going on when I didn't know about this.